### PR TITLE
Blast Menu UI Tweaks

### PR DESCRIPTION
**Pull request by justinmc**
Issue: https://github.com/Autodesk/genetic-constructor/issues/1758

This PR makes a few tweaks to the plugins context menu to make it match the spec in [zeplin](https://app.zeplin.io/project.html#pid=581d0bc0b7502238616341cb&sid=58a4ea278d3c3a5d387d7986).

screen shot 2017-04-13 at 2 22 35 pm

### CODE REVIEW

**Comment by justinmc** [comment]
Sorry, this isn't quite ready for review yet.  I didn't realize mentionbot was going to jump in and tag people :)

File: src/components/Menu/MenuItem.js [hunk 1]
```diff
@@ -34,13 +34,18 @@ export default class MenuItem extends Component {
     action: PropTypes.func,
     disabled: PropTypes.bool,
     checked: PropTypes.bool,
+    indent: PropTypes.bool,
```
Comment (justinmc) [inline src/components/Menu/MenuItem.js]: Are we ok with adding an indent prop here?  Any better way to do this?

File: src/graphics/views/constructviewer.js [hunk 2]
```diff
@@ -417,11 +417,15 @@ export class ConstructViewer extends Component {
 
     const extensionsWithBlockMenus = extensionsByRegion('menu:block')
     .filter(manifest => manifest.render['menu:block'])
-    .map(manifest => manifest.render['menu:block'](singleBlock, firstBlock));
+    .map(manifest =>
+      manifest.render['menu:block'](singleBlock, firstBlock)
+      // Indent all extension menuItems
+      .map(menuItem => Object.assign({}, menuItem, { indent: true })),
```
Comment (justinmc) [inline src/graphics/views/constructviewer.js]: I add `intent: true` to all extension menuItems, which I thought was more robust than just adding it to [extensions/blast/blockMenuItem.js](https://github.com/Autodesk/genetic-constructor/blob/master/extensions/blast/blockMenuItem.js).  Let me know if that's not true or if there's a better place to do this.

Comment (justinmc) [inline src/graphics/views/constructviewer.js]: I really wanted to add a quick spec for this just to check that all extension menu items have `indent: true`, but it looks like that's going to be a big undertaking.  This component doesn't have a spec file yet, and I would have to write a fake store factory or something like that to get it to render in a test.

I could split this `blockContextMenuItems` method out into a utils file or something if we want, that would be much more testable.

Comment (maxbates) [inline src/graphics/views/constructviewer.js]: none of the components have tests right now.... they are just implicitly tested by the end to end tests

**Pull request merged**